### PR TITLE
Add reflexive relationships to road-sign-concept model

### PR DESCRIPTION
**Pull request by anitacaron**
And also an example of how to get the sub sign and related road sign data

### CODE REVIEW

File: app/templates/road-sign-concepts/road-sign-concept.hbs [hunk 1]
```diff
@@ -10,27 +10,178 @@
...
+  <AuToolbarGroup>
+    <AuHeading @skin="2">
+      Sub Signs
+    </AuHeading>
+  </AuToolbarGroup>
+</AuToolbar>
+
+<AuToolbar @size="large">
+  <AuToolbarGroup>
+    {{#each @model.subSigns as |subSign|}}
```
Comment (anitacaron) [inline app/templates/road-sign-concepts/road-sign-concept.hbs:75]: No need to approve this one, just to show how to use it.

**Comment by Windvis** [comment]
The model changes look good to me, but I would split them off from the template rework? Maybe you can delete the models we no longer need as well then?

**Comment by anitacaron** [comment]
Yes, I'll delete them from the template.

After this, I can delete the models we're not using anymore.

**Comment by Windvis** [comment]
Can you rebase on the main branch (or merge the main branch in yours)? It seems this branch is 15 commits behind.

**Comment by anitacaron** [comment]
I think we can close this PR. I'll create a branch from the main and work from there to insert the edit sub sign functionality.

**Pull request closed without merging**